package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "autoprefixer": "^10.4.16",
-    "mongodb": "^6.1.0",
```

Comment:
Any reason why you downgraded Mongodb? @Mpho-vincent-makgwe

Reply:
forgot to upgrade it back..I was trying to get the length of recipes from mongo